sirepo/template/srw.py

```diff
@@ -2270,6 +2250,38 @@ def _wavefront_pickle_filename(el_id):
     return 'initial.pkl'
 
 
+def _write_rsopt_zip(data, ctx):
+    filename = f'{ctx.fileBase}.zip'
```

Comment:
Add this to  _rsopt_jinja_context() so not needed as a constant here.

Reply:
[210b274](https://github.com/radiasoft/sirepo/pull/4057/commits/210b27494aeea9efe147b4e3ea574847edabba0d)